docs/developers/devel-env.rst

```diff
@@ -69,6 +69,7 @@ configuration.
 
     cp listenbrainz/config.py.sample listenbrainz/config.py
 
+Don't change anything in config.py.sample.
```

Comment:
We can even lay it thicker I think:
```suggestion
This will create a config.py file. Don't change anything in config.py.sample: as its name indicates it is only a sample file.
```

And the following sentence which tripped you up initially:
```suggestion
Next, open the new config.py file with your favorite text editor
```

Reply:
I actually like the brief instructions without any fluff, I don’t think it needs that extra line. What tripped me up is that ‘sample’ is mentioned earlier in that part:
“If this is your first time configuring ListenBrainz, copy the sample to a live configuration.”
I thought changing both would be a safe bet in case the rest of the section was talking about ‘sample’ stuff.

How about ’Now, open the new config.py file (don’t change config.py.sample) with your favorite text editor’?

’New’ implies that the config.py file was created by the previous step without having to go into it too much.

Reply:
Yes, I think that will work absolutely fine